ts/store/reducers/backendInfo.ts

```diff
@@ -40,3 +41,7 @@ export default function backendInfo(
       return state;
   }
 }
+
+// Selectors
+export const serverInfoData = (state: GlobalState) =>
```

Comment:
this is not a selector. Try to use [createSelector](https://github.com/reduxjs/reselect) instead

Reply:
@Undermaken I saw that a 'simple' selector is created here too, is it wrong? https://github.com/teamdigitale/io-app/blob/91aad8da5a017389bf6b1428e102b7813918949c/ts/store/reducers/entities/services/index.ts#L73